# this commands joint accelerations

int16 joint1
int16 joint2
